color_helper.py

```diff
@@ -72,6 +72,9 @@ def start_file_index(view):
                     if s.get('show_index_status', True):
                         sublime.status_message('File color indexer started...')
 
+def preview_is_on_left():
+    """Return boolean for positioning preview on left/right"""
```

Comment:
This has already been resolved in https://github.com/facelessuser/ColorHelper/pull/65/commits/2c99c819a317ab09c20522faca53308694e2ea09 . Travis CI caught it and I patched it =/